sparktk-core/src/main/scala/org/trustedanalytics/sparktk/models/classification/naive_bayes/NaiveBayesModel.scala

```diff
@@ -105,7 +105,7 @@ case class NaiveBayesModel private[naive_bayes] (sparkModel: SparkNaiveBayesMode
    * @param columns Column(s) containing the observations whose labels are to be predicted.
    *                By default, we predict the labels over columns the NaiveBayesModel
    */
-  def predict(frame: Frame, columns: Option[List[String]] = None): Unit = {
+  def predict(frame: Frame, columns: Option[List[String]] = None): Frame = {
```

Comment:
Also, update function description, since right now it's saying that it "adds a column" to the frame